pyartcd/pyartcd/pipelines/gen_assembly.py

```diff
@@ -129,27 +130,14 @@ async def run(self):
             await self._slack_client.say(f"Error generating assembly definition for {self.assembly}", slack_thread)
             raise
 
-    async def _get_latest_accepted_nightly(self):
+    def _get_latest_accepted_nightly(self):
         self._logger.info('Retrieving most recent accepted amd64 nightly...')
-
         major, minor = isolate_major_minor_in_group(self.group)
-        tag_base = f'{major}.{minor}.0-0.nightly'
-        rc_endpoint = f"{rc_api_url(tag_base, 'amd64', self.private_nightlies)}/tags"
-
-        async with aiohttp.ClientSession() as session:
-            async with session.get(rc_endpoint) as response:
-                if response.status != 200:
-                    self._logger.warning('Failed retrieving latest accepted nighly from %s', rc_endpoint)
-                    return None
-
-                tags = (await response.json()).get('tags', [])
-                accepted_nightlies = list(filter(lambda tag: tag['phase'] == 'Accepted', tags))
-
-                if accepted_nightlies:
-                    return accepted_nightlies[0]['name']
-                else:
-                    self._logger.warning('No accepted nightly found')
-                    return None
+        latest_accepted_nightly = requests.get(f"{constants.RC_AMD64_URL}/{major}.{minor}.0-0.nightly/latest")
```

Comment:
`rc_api_url` should be preferred to retrieve the endpoint URL.

Reply:
updated in https://github.com/openshift-eng/art-tools/pull/1097/commits/b90c644869b508bc909274fb7c7c1f20e2f6c894